source/MaterialXCore/Node.cpp

```diff
@@ -267,12 +267,13 @@ void GraphElement::flattenSubgraphs(const string& target, NodePredicate filter)
 
             NodeGraphPtr sourceSubGraph = pair.second;
             std::unordered_map<NodePtr, NodePtr> subNodeMap;
-
-            // Create a new instance of each original subnode.
```

Comment:
Is this comment no longer accurate, or did it get deleted accidentally?

Reply:
Accident. Fixed now.